### DISCUSSION TITLE
Implementing case-insensitive for _ncontains filter rule

### DISCUSSION THREAD

**Discussion by rxob** [directus/directus#19265] [Feature Requests]
### Summary

There are two filter rules: _contains and _icontains. It will be a great idea to make that logic possible to _ncontains which at the moment is case-sensitive.  

### Motivation

Expected result is to retrieve items that don't contain a string regardless of its form

### Detailed Design

New filter rule similar to _ncontains but which will be case-insensitive.

### Must Have:
- creating additional filter rules to match case-insensitive. A good name can be:  _nicontains

### Drawbacks

There is an alternative logic and you can achieve a similar result

### Alternatives

You can achieve a similar result by filtering items that have you specific string with _icontains and taking their ids. then make another request to get all items that don't have a specific id from prev result.

### Adoption Strategy

This will not affect other projects because this feature will only add a new filter rule to existing ones

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
